Dockerfile

```diff
@@ -73,15 +73,27 @@ RUN yum update -y
 # to use in runtime the git version installed by lambda-git package
 RUN yum install git -y
 
-COPY requirements.txt ./
+COPY requirements.in ./
```

Comment:
I think the problem is with using `.in` is that the versions aren't pinned.  So we will get a new version at each time this `Dockerfile` is ran. I do not recommend this approach.

Reply:
thanks @xbeta, trying another approach now